packages/editor-ui/src/components/OutputPanel.vue

```diff
@@ -277,6 +283,14 @@ watch(defaultOutputMode, (newValue: OutputType, oldValue: OutputType) => {
 const activatePane = () => {
 	emit('activatePane');
 };
+
+const shouldShowWarning = computed(() => {
+	if (!node.value) return false;
+
+	const parents = props.workflow.getParentNodes(node.value.name, NodeConnectionType.AiTool, 1);
+	const active = parents.filter((x) => !!workflowRunData.value?.[x]?.[props.runIndex]);
```

Comment:
I would also suggest a more specific variable name here since `active` is quite generic.